akka-projection-jdbc/src/main/scala/akka/projection/jdbc/javadsl/JdbcHandler.scala

```diff
@@ -18,12 +21,17 @@ object JdbcHandler {
    * INTERNAL API
    */
   @InternalApi
-  private class HandlerFunction[Envelope, S <: JdbcSession](handler: (S, Envelope) => Unit)
+  private class HandlerFunction[Envelope, S <: JdbcSession](handler: BiConsumer[S, Envelope])
       extends JdbcHandler[Envelope, S] {
-    override def process(session: S, envelope: Envelope): Unit = handler(session, envelope)
+    override def process(session: S, envelope: Envelope): Unit = handler.accept(session, envelope)
   }
 
+  @Deprecated
+  @deprecated("Use the similar method with Java functional interface as a parameter", "1.2.1")
   def fromFunction[Envelope, S <: JdbcSession](handler: (S, Envelope) => Unit): JdbcHandler[Envelope, S] =
+    new HandlerFunction(handler.asJava)
```

Comment:
I think this is simple enough to write without depending on the FunctionConverters:
```
new HandlerFunction((s, env) => handler(s, env))
```

Reply:
🆗